internal/service/cassandra/cassandra_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"regexp"
 	"testing"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
```

Comment:
should be grouped with other `github.com/hashicorp/terraform-plugin-sdk` imports